src/managers/GCommandLoader.js

```diff
@@ -71,9 +72,9 @@ class GCommandLoader {
             this.GCommandsClient.emit(Events.LOG, new Color(`&d[GCommands] &aLoaded (File): &e➜   &3${fileName}`, { json: false }).getText());
         }
 
-        await this.__loadSlashCommands();
```

Comment:
I think those awaits should be there, otherwise, the `COMMANDS_LOADED` event gets emitted early